pkg/eventtype/eventtypes.go

```diff
@@ -93,11 +93,14 @@ func (h *EventTypeAutoHandler) AutoCreateEventType(ctx context.Context, event *e
 					},
 				},
 			},
-			Spec: v1beta2.EventTypeSpec{
-				Type:        event.Type(),
-				Source:      source,
-				Schema:      schema,
-				SchemaData:  event.DataSchema(),
+			Spec: v1beta3.EventTypeSpec{
+				Attributes: []v1beta3.EventAttributeDefinition{
+					{Name: "type", Value: event.Type(), Required: true},
+					{Name: "source", Value: source.String(), Required: true},
+					{Name: "schemadata", Value: schema.String(), Required: true},
+					{Name: "specversion", Value: event.SpecVersion(), Required: true},
+					{Name: "id", Required: false},
```

Comment:
the CE spec does require it...

Reply:
Ah my bad, I'll open a quick follow up :)